.github/workflows/node.js.yml

```diff
@@ -10,31 +7,48 @@ on:
   pull_request:
 
 jobs:
-  build:
+  test:
+    name: Run tests on supported Node.js versions
     runs-on: ubuntu-latest
 
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 17.x, 18.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        # See supported Node.js versions at https://nodejs.org/en/about/previous-releases
+        node-version: [18, 20, 21]
 
     steps:
-    - uses: actions/checkout@main
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@main
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm test
+      - name: Check out repository
+        uses: actions/checkout@v4
```

Comment:
All actions now use the latest version in the major version range, this should prevent any accidental regressions.